hmda-analytics/src/main/scala/hmda/analytics/HmdaAnalyticsApp.scala

```diff
@@ -428,37 +438,42 @@ object HmdaAnalyticsApp extends App with TransmittalSheetComponent with LarCompo
 
         _ <- deleteTsRow
         _ = if(tsDeletion)
-          log.info(s"Data removed from TS for  $submissionId ")
+          log.info(s"Attempt to remove data from TS table for  $submissionId  completed.")
         else
           log.info(s"Skipping Delete TS -- no deletion needed")
```

Comment:
I feel like these logging messages should really be within the processing methods instead of a standalone block here, i.e. putting them in `deleteTsRow`, `deleteLarRows`, etc... thoughts?

Reply:
or these can be something more generic like `deleteTsRow checkpoint`, but the actual informative messages be inside the process methods.

Reply:
i'll make these changes  this evening @lchen-2101